### ISSUE TITLE
[BUG] Private build JARs checked into the tree for BWC tests

### ISSUE THREAD

**Issue by dblock** [opensearch-project/anomaly-detection#352]
**Describe the bug**

Currently the following JARs are checked into the tree. These are private builds of job-scheduler.

![Screen Shot 2022-01-07 at 2 41 24 PM](https://user-images.githubusercontent.com/542335/148598153-f37f513d-7531-497d-8b0f-a8f55c14b544.png)

**Expected behavior**

No private build JARs checked in, fetch from https://aws.oss.sonatype.org/ via a dependency.

**Comment by amitgalitz** [comment]
As of right now it seems like the snapshot for 1.3.0 isn't available on maven. Reaching out to job-scheduler/infra to make sure that gets updated and then will move our workflow to use a snapshot instead

**Comment by dblock** [comment]
I see 1.3.0 in https://aws.oss.sonatype.org/#nexus-search;gav~org.opensearch~opensearch~~~~kw,versionexpand

**Comment by amitgalitz** [comment]
> [BUG] Checked in versions of job-scheduler-*-SNAPSHOT.zip #17

Yes, only spi is up to date not job-scheduler. Thanks for closing this one.
